### ISSUE TITLE
metrics/view archive link doesnt appear on openshift 3.9 webconsole

### ISSUE THREAD

**Issue by SubhakarKotta** [gshipley/installcentos#64]
I have installed openshift-3.9 version and it doesnt show any view arhive/metrics link on the UI web console

**Comment by marekjelen** [comment]
How many resources were dedicated to the machine OpenShift runs on?

**Comment by SubhakarKotta** [comment]
4 cpu's and 20 GB memory.

**Comment by ryannix123** [comment]
That should be more than enough for metrics and logging. Are you certain ports 80, 443, and 8443 are open to the world?

**Comment by SubhakarKotta** [comment]
Yes they are open. Now I am again trying to reinstall again and let you know is that working...
